docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md

```diff
@@ -18,7 +18,13 @@ driver.get("http://www.google.com");
 driver.quit();
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# We don't have a Python code sample yet -  Help us out and raise a PR  
+ from selenium import webdriver
+ from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
```

Comment:
Please do not use DesiredCapabilities, you can follow the other examples and use FirefoxOptions